packages/amplify-graphql-api-construct/src/internal/data-source-config.ts

```diff
@@ -93,3 +99,82 @@ export const getDataSourceStrategiesProvider = (definition: IAmplifyGraphqlDefin
 
   return provider;
 };
+
+/**
+ * Creates a new schema by merging the individual schemas contained in the definitions, combining fields of the Query and Mutation types in
+ * individual definitions into a single combined definition. Adding directives to `Query` and `Mutation` types participating in a
+ * combination is not supported (the behavior is undefined whether those directives are migrated).
+ */
```

Comment:
When merging schemas, what happens if there are conflicts. Should that be called out in the docstring here? Is the behavior defined in customer-facing docs?